indico/modules/groups/controllers.py

```diff
@@ -31,7 +32,16 @@
 from indico.web.rh import RHProtected
 
 
-class RHGroups(RHAdminBase):
+class RHLocalGroupsBase(RHAdminBase):
+    """Class for checking Local group configuration"""
+
+    def _check_access(self):
+        RHAdminBase._check_access(self)
+        if not config.LOCAL_GROUPS:
+            raise Forbidden('Local groups are disabled.')
+
+
+class RHGroups(RHLocalGroupsBase):
```

Comment:
I just realized that this page is not only for local groups but all types of groups. So disabling everything through inheritance won't work... Sorry for that!

So the check needs to be in `RHGroupBase`, by checking if `group.is_local` there. and `RHLocalGroupBase` (unconditionally since that one is only used for local ones).